.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!--
+Have you read Pulsar's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/pulsar-edit/.github/blob/master/CODE_OF_CONDUCT.md
+Do you want to ask a question? Are you looking for support? The Pulsar Discussion tab is the best place for getting support: https://github.com/orgs/pulsar-edit/discussions
+-->
+
+### Prerequisites
+
+* [ ] Put an X between the brackets on this line if you have done all of the following:
+    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
+    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
+    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
+    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Apulsar>
```

Comment:
hmm, this link doesn't exactly look right?

Reply:
Changed the wording to look like:

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Have you reproduced the issue in safe mode? : <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Have you used the debugging guide to fix any issues? : <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked our FAQs to make sure your question isn't answered there? : <https://discuss.atom.io/c/faq>
    *Does your issue already exist? : <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Apulsar>
    * Checked that there is not already an Pulsar package that provides the described functionality: <https://pulsar-edit.com/packages>